src/MagnumPlugins/ObjImporter/Test/Test.cpp

```diff
@@ -406,7 +427,6 @@ void ObjImporterTest::wrongFloat() {
 
     std::ostringstream out;
     Error redirectError{&out};
-    CORRADE_VERIFY(!importer.mesh3D(id));
```

Comment:
I was actually using CORRADE_SKIP alot in the AssimpImporter, to also provide the reason why it fails, no idea what I meant to do here... :thinking: